pkg/apis/serving/v1alpha1/revision_lifecycle.go

```diff
@@ -45,9 +45,13 @@ const (
 	RequestQueueAdminPortName string = "queueadm-port"
 
 	// RequestQueueMetricsPortName specifies the port name to use for metrics
-	// emitted by queue-proxy.
+	// emitted by queue-proxy for autoscaler.
 	RequestQueueMetricsPortName = "queue-metrics"
```

Comment:
nit: can we rename `RequestQueueMetricsPortName` to `InternalQueueMetricsPortName` or `AutoscalingQueueMetricsPortName` or something similar?

Reply:
Will do.